README.md

```diff
@@ -82,27 +73,17 @@ module ApiDefinitions
   end
 end
 ```
-
 Now, if you access: [http://localhost:3000/adder.json?a=4](http://localhost:3000/adder.json?a=4) you should see the result:
 
-```json
-{
-  "result": 4
-}
-```
+    {
+    "result": 4
+    }
 
 and similarly [http://localhost:3000/adder.json?a=4&b=22](http://localhost:3000/adder.json?a=4&b=22) (etc)
 
+## Advanced
 
-## Contributing
-
-### Documentation
-
-Use [yard](https://rubydoc.info/gems/yard/file/README.md) to update the gem documentation. Have a
-look at the [yard tags docs](https://rubydoc.info/gems/yard/file/docs/Tags.md) for how to use tags.
-
-```sh
-yard
-```
-
-`yard` itself is configured by `.yardopts`.
+- [Implementing a POST endpoint](file.POST.html)
```

Comment:
Those links don't work. In GitHub those are resolved as `https://github.com/sharesight/flappi/blob/ft_moar_docz/file.POST.html` which doesn't exist.

Reply:
Because the HTML isn't deployed - when this is merged to master, they will be correct as part of the github page:
http://sharesight.github.com/flappi